test/e2e/workflow/validation_test.go

```diff
@@ -39,7 +39,7 @@ var _ = Describe("NetworkAddonsConfig", func() {
 	Context("when a valid config is deployed", func() {
 		BeforeEach(func() {
 			configSpec := opv1alpha1.NetworkAddonsConfigSpec{
-				LinuxBridge: &opv1alpha1.LinuxBridge{},
+				ImagePullPolicy: v1.PullAlways,
```

Comment:
then i will remove the test totally because i can revert to linuxbridge as it now supports removal, ok ?
(btw this is the failing test that doesnt fail on master, maybe just this one is the leftover)